src/filter.rs

```diff
@@ -231,25 +217,23 @@ pub fn evaluate(
                         return true;
                     }
                 }
-                xop => {
+                _ => {
```

Comment:
here it'd be nice to have a `info!` or `warn!` that shows an unimplemented operation is being used, such as `<` or `>`  which we don't support right now

Reply:
I added that on the other PR